lib/search.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const request = require('./utils/request');
+const appData = require('./app');
```

Comment:
one last thing, by requiring the appMethod here, you may be loosing the [memoization option](https://github.com/facundoolano/google-play-scraper#memoization) when it's set.

Instead of requiring you should pass the app function as an argument in index.js, like getParseList is passed now

Reply:
Oh, thats actually something I haven't even thought on.
Going change this.
Thanks